Cocktails9/Cocktails9/View/BottomRow/CustomTextFieldStyle.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+struct CustomTextFieldStyle: ViewModifier {
```

Comment:
This is ok to be done as ViewModifier, but keep in mind there is specialized TextFieldStyle instead of it